scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -119,13 +120,13 @@ purrr::map(squares, # vector to map over
 
 The output from running `purr::map()` is always a list (but note that there are other `purrr::map()` relatives which return other object types, as you can read about in [the `purrr::map()` documentation](https://purrr.tidyverse.org/reference/index.html)).
 
-We'll note that, in this case, it's possible to simply run `sqrt()` directly on the `squares` vector itself:
+We'll note that, in this case, it's possible to simply run `sqrt()` directly on the `squares` vector itself since this function is already vectorized, but not all functions behave this way.
 
-```{r quick squares}
+```{r quick_squares}
```

Comment:
We have not been doing space-free names in other notebooks (at least not in this module), so for consistency I wouldn't make this change.

Reply:
I'll revert throughout.